charlib/characterizer/procedures/combinational/delay.py

```
@@ -178,14 +185,21 @@ def measure_delays_for_path_with_criterion(cell, config, settings, variation, pa
         # Build LUT
-        delay_measurements =[analysis.measurements[name] for analysis in analyses.values() if name in analysis.measurements]
-        delay = criterion(delay_measurements) @ PySpice.Unit.u_s
+        delay_measurements = [analysis.measurements[name] for analysis in analyses.values() if name in analysis.measurements]
+        try:
+            delay = criterion(delay_measurements) @ PySpice.Unit.u_s
+        except ValueError as e:
+            if settings.dry_run:
+                delay = -1 @ PySpice.Unit.u_s
+            else:
+                raise
         lut_name, meas_path = name.split('__')
```

Ruff (0.15.21)

[warning] Unpacked variable `meas_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)